README.md

```diff
@@ -1,22 +1,24 @@
-# kube-compare
+# cluster-compare
```

Comment:
It's a bit confusing. The repo name is `kube-compare` and here we are calling it `cluster-compare`

Reply:
I think it's usually the name of the repo

Reply:
Put that back.